src/components/templates/chat/create-group-chat.jsx

```diff
@@ -84,16 +84,18 @@ const CreateGroupChat = () => {
                   <label className="text-xs text-slate-200">
                     Select a user
                   </label>
-                  <select
-                    placeholder="Select a user..."
-                    autoComplete="false"
-                    className="w-full border-[1px] border-white bg-black p-4 text-white placeholder:text-gray-500"
-                  >
-                    <option value="0">John Doe</option>
-                    <option value="1">Jane Doe</option>
-                    <option value="2">Mark Smith</option>
-                    <option value="3">Tiger Woods</option>
-                  </select>
+                  <div className="w-full border-[1px] border-white px-4">
```

Comment:
Only giving padding at the right side makes sense in my opinion. Giving left side padding to the select pushes selected option text to the right, creating inconsistency. Please replace `px-4` to `pr-4`

Reply:
I think this is now ok! 😅 

![image](https://github.com/hiteshchoudhary/devui/assets/99522246/18fd2944-5351-4af3-9a30-f4b0d3298816)

Reply:
I have resolved it and made **pr-4 to the outer div** and made **checkbox pl-4** (so that text get a little breath with pl-4)